pkg/sfu/downtrack.go

```diff
@@ -483,6 +490,26 @@ func (d *DownTrack) WritePaddingRTP(bytesToSend int) int {
 	return bytesSent
 }
 
+func (d *DownTrack) updateBitrate() {
+	lastRtp := d.lastRTP.Load()
+	d.statsLock.RLock()
+	update := lastRtp.Sub(d.lastBitrateReport) > time.Second
+	d.statsLock.RUnlock()
+	if !update {
+		return
+	}
+	octets, _ := d.getSRStats()
+	d.statsLock.Lock()
+	d.bitrate = octets*8 - d.bitrateHelper
```

Comment:
This might be okay, but to be accurate, this should take the time difference and scale like `sfu.buffer` does.